### PR TITLE
Added proc module for HA config

### PR DESCRIPTION
**Pull request by csibbitt**
Anyone else have opinions on what else could be added here? Seems short and sweet.

This content hasn't been linked into the rest of the docs yet because I expect a pretty major shuffle when we start the refactoring of the docs for GA during next sprint. Not 100% sure where it will fit best.

### CODE REVIEW

**Comment by csibbitt** [comment]
# Testing
I hacked it into one of the existing assemblies just to test the rendering. Here is a screenshot of the text just to ensure my tokens got replaced sensically. (Note that "OKD" is currently the correct replacement, though I don't think we've actually tested any of the new stuff with OKD afaik)

image

**Comment by csibbitt** [comment]
[Documentation updates for HA QDR](https://trello.com/c/9vdW5jXk/787-documentation-updates-for-ha-qdr)

File: doc-Service-Telemetry-Framework/modules/proc_configuring-high-availability.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// The `context` attribute enables module reuse. Every module's ID includes
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="configuring-high-availability_{context}"]
+= Configuring High Availability
+
+To configure {Project} for High Availability, the `highAvailabilityEnabled: true` attribute must be added to the servicetelemetry object in {OpenShiftShort}.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_configuring-high-availability.adoc]: `ServiceTelemetry` object right?

File: doc-Service-Telemetry-Framework/modules/proc_configuring-high-availability.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+This can either be set at installation time, or if you already have {ProjectShort} deployed, you can follow this procedure:
+
+. Use the oc command to edit the servicetelemetry object:
++
+----
+    oc edit servicetelemetry
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_configuring-high-availability.adoc]: Now that I've looked here, I see what you mean about `servicetelemetry` above. Does this actually work? I think you need to provide it an actual object to edit of type `servicetelemtry` right?

e.g. `oc edit servicetelemetrys.infra.watch stf-default`

Comment (csibbitt) [reply doc-Service-Telemetry-Framework/modules/proc_configuring-high-availability.adoc]: It does work, yes

**Comment by csibbitt** [comment]
# Additional testing

## Bare unqualified type with no named object
```
$ oc edit servicetelemetry

# Please edit the object below. Lines beginning with a '#' will be ignored,
# and an empty file will abort the edit. If an error occurs while saving this file will be
# reopened with the relevant failures.
#
apiVersion: infra.watch/v1alpha1
kind: ServiceTelemetry
metadata:
  creationTimestamp: "2020-02-25T22:26:36Z"
  generation: 1
  name: stf-default
[...]
```

## Multiple objects of the same type
```
$ oc edit interconnect

# Please edit the object below. Lines beginning with a '#' will be ignored,
# and an empty file will abort the edit. If an error occurs while saving this file will be
# reopened with the relevant failures.
#
apiVersion: v1
items:
- apiVersion: interconnectedcloud.github.io/v1alpha1
  kind: Interconnect
  metadata:
    creationTimestamp: "2020-02-25T23:58:32Z"
    generation: 2
    name: qdr-test
[...]
- apiVersion: interconnectedcloud.github.io/v1alpha1
  kind: Interconnect
  metadata:
    creationTimestamp: "2020-02-25T22:26:55Z"
    generation: 1
    name: stf-default-interconnect
```

File: doc-Service-Telemetry-Framework/modules/proc_configuring-high-availability.adoc (hunk 1 above)
Comment (csibbitt) [reply doc-Service-Telemetry-Framework/modules/proc_configuring-high-availability.adoc]: Sure. That version also tests okay; I promise!

**Comment by JoanneOFlynn2018** [comment]
> Looks awesome. Technical review approved. Ready to tag Joanne.

Is this ready for me now?

**Comment by csibbitt** [comment]
@JoanneOFlynn2018  Yes. As noted I wasn't sure how this would fit into the re-organized document, but this is the content we need.

**Pull request merged**